Tests/SotoTests/Services/S3/S3ExtensionTests.swift

```diff
@@ -386,84 +237,85 @@ class S3ExtensionTests: XCTestCase {
-                return s3.putObject(putRequest, on: runOnEventLoop)
-            }
-            .flatMap { _ -> EventLoopFuture<S3.SelectObjectContentOutput> in
-                let expression = "Select * from S3Object"
-                let input = S3.InputSerialization(csv: .init(fieldDelimiter: ",", fileHeaderInfo: .use, recordDelimiter: "\n"))
-                let output = S3.OutputSerialization(csv: .init(fieldDelimiter: ",", recordDelimiter: "\n"))
-                let request = S3.SelectObjectContentRequest(
-                    bucket: name,
-                    expression: expression,
-                    expressionType: .sql,
-                    inputSerialization: input,
-                    key: "file.csv",
-                    outputSerialization: output,
-                    requestProgress: S3.RequestProgress(enabled: true)
-                )
-                let size = file10.utf8.count
-                var returnedSize = 0
-                return s3.selectObjectContentEventStream(request, logger: TestEnvironment.logger, on: runOnEventLoop) { eventStream, eventLoop in
-                    XCTAssertTrue(eventLoop === runOnEventLoop)
-                    switch eventStream {
-                    case .records(let records):
-                        if let payload = records.payload {
-                            if let decodedCount = payload.decoded()?.count {
-                                returnedSize += decodedCount
-                                print("Record size: \(decodedCount)")
-                            } else {
-                                XCTFail("Failed to decode Base64 data in payload")
-                            }
-                        }
-                    case .stats(let stats):
-                        if let details = stats.details {
-                            print("Stats: ")
-                            print("  processed: \(details.bytesProcessed ?? 0)")
-                            print("  returned: \(details.bytesReturned ?? 0)")
-                            print("  scanned: \(details.bytesScanned ?? 0)")
-
-                            XCTAssertEqual(Int64(size), details.bytesProcessed)
-                            XCTAssertEqual(Int64(returnedSize), details.bytesReturned)
-                        }
-                    case .end:
-                        print("End")
-                    default:
-                        break
-                    }
-                    return eventLoop.makeSucceededFuture(())
-                }
-            }
-            .flatAlways { _ in
-                return S3Tests.deleteBucket(name: name, s3: s3)
-            }
-        XCTAssertNoThrow(try response.wait())
-    }
-
-    func testS3VirtualAddressing(_ urlString: String, config: AWSServiceConfig = S3ExtensionTests.s3.config) throws -> String {
+    /*
+     func testSelectObjectContent() throws {
```

Comment:
Is this to be brought back in a future PR?

Reply:
yes